storybook/stories/API/chart/SunburstChart.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { ResponsiveContainer, SunburstChart, Tooltip } from '../../../../src';
+import { CategoricalChartProps } from '../props/ChartProps';
+import { hierarchy } from '../../data';
+
+export default {
+  argTypes: {
+    ...CategoricalChartProps,
```

Comment:
This seems to lead to misleading descriptions. Instead of importing this group, we should only import the specific shared descriptions that are actually used. 

Otherwise, props such as barCategoryGap show up, which are not used.

Reply:
<img width="815" alt="Screenshot 2024-01-07 at 20 54 22" src="https://github.com/recharts/recharts/assets/27995606/3746211c-c6d2-49d4-905c-378de078634f">